utils/manage_static.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+# -*- coding: utf-8; mode: sh indent-tabs-mode: nil -*-
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+BUILD_COMMIT_MESSAGE="Static build"
```

Comment:
Is it OK when we use 

    [build] /static

Reply:
I'm not sure to understand. 
Whatever the name, we just need one.

Reply:
> Whatever the name, we just need one.

OK, then lets use `[build] /static`

> I'm not sure to understand.

This PR is for _manage_static_ files we generate in the static folder ... may be later, we add a _manage_data_ script which the can use `[build] /data` .. so we have commit massages with canonical names.